sync-service/src/main/java/com/pavax/stayhome/syncservice/service/BusinessEntryDto.java

```diff
@@ -13,7 +13,6 @@
 	@NotBlank
 	private String description;
 
-	@Size(max = 255)
```

Comment:
@mageo  FYI: removed the constraint on my side as you changed that field to a 'text' field on your side